cupy_backends/hip/cupy_hipsolver.h

```diff
@@ -17,9 +17,8 @@ hipsolverStatus_t cusolverGetProperty(hipLibraryPropertyType_t type, int* val) {
     return HIPSOLVER_STATUS_SUCCESS;
 }
 
-typedef enum hipsolverDnParams_t {};
 
-#if HIP_VERSION < 50631061
+#if HIP_VERSION < 60240094
```

Comment:
Double check this one again, same as above.

Reply:
Addressed this above